Update lambda manager to add permissions using wildcard (#178)
coralogix/coralogix-aws-serverless: src/lambda-manager/README.md

| DESTINATION_ROLE | Arn for the role to allow destination subscription to be pushed (needed only for Firehose) | | :heavy_check_mark: |
| DISABLE_ADD_PERMISSION | Skip Adding LogGroup Permission for lambda | false | |
| DESTINATION_TYPE | Type of destination (Lambda or Firehose) | | :heavy_check_mark: |

> | SCAN_OLD_LOGGROUPS | When set to true, the lambda will scan all existing log group and add the ones that match the RegexPattern as a trigger, the scan will only happen on the creation of the lambda after that it will only detect a new log group. | false | |

Reviewer: ```suggestion
| SCAN_OLD_LOGGROUPS | When true, the Lambda scans all existing log groups on creation and adds those matching RegexPattern as triggers. After creation, only new log groups are detected. | false | |
```

> | ADD_PERMISSIONS_TO_ALL_LOG_GROUPS | When set to true, the code will allow all existing and new log groups using a wildcard to subscribe to the destination. | false | |

Reviewer: ```suggestion
| ADD_PERMISSIONS_TO_ALL_LOG_GROUPS | When true, grants subscription permissions to all current and future log groups using a wildcard. | false | |
```
